dist/common/systemd/scylla-server.slice

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Slice used to run Scylla. Maximum priority for IO and CPU
+Before=slices.target
+
+[Slice]
+MemoryAccounting=true
+IOAccounting=true
+CPUAccounting=true
+# Systemd deprecated settings BlockIOWeight and CPUShares. But they are still the ones used in RHEL7
+# Newer SystemD wants IOWeight and CPUWeight instead. Luckily both newer and older SystemD seem to
+# ignore the unwanted option so safest to get both. Using just the old versions would work too but
+# seems less future proof. Using just the new versions does not work at all for RHEL7/
+BlockIOWeight=1000
+IOWeight=1000
+# Should be zero, but work around https://github.com/systemd/systemd/issues/8363 for older systemd
+MemorySwapMax=1
+CPUShares=1000
+CPUWeight=1000
+
```

Comment:
Please check (by inspection is sufficient) that systemd does not create the quota stuff by itself.

Reply:
I have verified by looking at the cgroups hierarchy that the property is not set, if it is not mentioned by the slice:

```
$ cat cpu.cfs_quota_us 
-1
$ pwd
/sys/fs/cgroup/cpu,cpuacct/scylla.slice
cd scylla-server.slice/
$ cat cpu.cfs_quota_us 
-1
```

After setting `CPUQuota` to something (not part of the patch, just making sure):

```
$ cat cpu.cfs_quota_us 
1000
```

Therefore, as expected, systemd is not messing with it.